src/less/less/mixins-common.less

```diff
@@ -1,13 +1,19 @@
 @import "variables-common.less";
 
 // base styles for BEM span.icon block
-.background-icon-base() {
+.background-icon-image-base() {
```

Comment:
Should we hide this away from our public API now? (just leaving the old one until v8?)

Reply:
Thoughts on the best way to "hide" it?

Reply:
Thoughts on the best way to "hide" it?

Reply:
Hmm, let's wait also. We also have plenty of time before 7.3.0 goes out I just realised. So we can do more refactoring as necessary.